ex6_3_3.py

```diff
@@ -0,0 +1,12 @@
+import string
+def count_word(text:str):
+    """
+    A function that gets one specific paragraph and has to return the length of each word.
+    :param text:one long paragraph.
+    :return:the length of each word in the text.
+    """
+
+
+    text=set(filter(lambda x: len(x) > 0,text.replace("\n"," ").translate(str.maketrans('', '')).split()))
```

Comment:
There are more elegant/readable ways of cleaning text from symbols, this line is hard to read.
Find a different way of doing this.